frame/ranked-collective/src/lib.rs

```diff
@@ -538,7 +545,7 @@ pub mod pallet {
 			who: AccountIdLookupOf<T>,
 			min_rank: Rank,
 		) -> DispatchResultWithPostInfo {
-			let max_rank = T::DemoteOrigin::ensure_origin(origin)?;
+			let max_rank = T::RemoveOrigin::ensure_origin(origin)?;
```

Comment:
The function doc mentions `AdminOrigin`.

Reply:
Done.